sdk/dotnet/Network/GetGatewayConnection.cs

```diff
@@ -128,7 +128,7 @@ public sealed class GetGatewayConnectionResult
         /// A mapping of tags to assign to the resource.
         /// </summary>
         public readonly ImmutableDictionary<string, string> Tags;
-        public readonly Outputs.GetGatewayConnectionTrafficSelectorPolicyResult TrafficSelectorPolicy;
+        public readonly ImmutableArray<Outputs.GetGatewayConnectionTrafficSelectorPolicyResult> TrafficSelectorPolicies;
```

Comment:
Looks like a breaking change. Can we MaxItems=1 it?

Reply:
Fixed!